app/forms/openid_connect_logout_form.rb

```diff
@@ -86,22 +86,32 @@ def load_identity
     if reject_id_token_hint?
       identity_from_client_id
     else
+      identity_from_token_hint(id_token_hint) || identity_from_client_id
+    end
+  end
+
+  def identity_from_token_hint(id_token_hint)
+    return nil if id_token_hint.blank?
+    payload, _headers = nil
+
+    _matching_cert = [
+      AppArtifacts.store.oidc_primary_public_key,
+      AppArtifacts.store.oidc_secondary_public_key,
+    ].compact.find do |key|
```

Comment:
having an unused variable here has me 👀 

would it make sense to break this into two methods, or to just ditch the variable entirely? Or switch to `.each` with a `return`?

Reply:
Ah yeah, I don't think the assignment is doing anything. I was originally considering using the matching cert to log which key was used, but I decided against it since logout is deprecated. I removed the assignment.